### PR TITLE
Made WebServiceInterface port configurable in interface config

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

File: lymph/web/interfaces.py [hunk 1]
```diff
@@ -42,12 +46,11 @@ def on_start(self):
...
         except SocketNotCreated:
-            socket = create_socket('%s:%s' % (self.config.get('ip') or
-                                              self.container.server.ip,
-                                              self.http_port),
-                                   inheritable=True)
-            socket_fd = socket.fileno()
-        self.http_socket = create_socket('fd://%s' % socket_fd)
+            logger.warning("socket for port %s wasn't created by node, binding from instance instead", self.http_port)
+            address = '%s:%s' % (self.container.server.ip, self.http_port)
+            self.http_socket = create_socket(address)
```
Comment (mouadino) [inline lymph/web/interfaces.py]: Why did you remove inheritable=True ?

Comment (mouadino) [inline lymph/web/interfaces.py]: Si I guess it wasn't needed at all, since this process is not suppose to spawn right ?

Comment (emulbreh) [inline lymph/web/interfaces.py]: yes, inheritable is only needed in the node process.

**Comment by mouadino** [comment]
LGTM
